newjersey/navigator.business.nj.gov
# Fixed extra webflow licenses

## Description

Somehow we had a bunch of extra licenses in webflow, turns out we didn't have a webflow delete licenses function. 



### Ticket



This pull request resolves [#187809299](https://www.pivotaltracker.com/story/show/187809299).

### Approach



### Steps to Test



Run the preview script added to package json and see that there are no files to be deleted. 

### Notes



## Code author checklist
